common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -199,6 +199,13 @@
     return get_value('SHOW_LANGUAGE_SELECTOR', settings.FEATURES.get('SHOW_LANGUAGE_SELECTOR', False))
 %></%def>
 
+<%def name="show_footer_language_selector(api_setting)"><%
```

Comment:
What is the use case for the ``api_setting`` parameter? My guess is that you want to provide this value when rendering the footer via the API (i.e. the marketing site) and that you want to use the setting when rendering the footer inline. My recommendation would be to have that logic in the clients that are rendering the template so then you wouldn't need this method at all. You would then leave the template rendering based on the context parameter.